niworkflows/utils/bids.py

```diff
@@ -151,11 +164,13 @@ def collect_participants(
 def collect_data(
     bids_dir,
     participant_label,
-    session_id=Query.OPTIONAL,
+    session_id=None,
     task=None,
     echo=None,
+    group_me=True,
```

Comment:
Non-obvious parameter name. And please add an entry to the docstring.

```suggestion
    group_multiecho=True,
```

Reply:
I believe I did that because the name overloads the function defined below, but I can modify the function name.

Reply:
Or `group_echos`?

Reply:
```suggestion
    group_echos=True,
```